{
	"id": "CVE-2021-33390",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2021-33390",
				"description": "dpic 2021.04.10 has a use-after-free in thedeletestringbox() function in dpic.y. A different vulnerablility than CVE-2021-32421.",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "nvd@nist.gov",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
							"base_score": 9.8,
							"base_severity": "CRITICAL",
							"temporal_score": 9.8,
							"temporal_severity": "CRITICAL",
							"environmental_score": 9.8,
							"environmental_severity": "CRITICAL"
						}
					}
				],
				"cwe": [
					{
						"source": "nvd@nist.gov",
						"cwe": [
							"CWE-416"
						]
					}
				],
				"references": [
					{
						"source": "cve@mitre.org",
						"url": "https://gitlab.com/aplevich/dpic/-/issues/10"
					},
					{
						"source": "nvd.nist.gov",
						"url": "https://nvd.nist.gov/vuln/detail/CVE-2021-33390"
					}
				],
				"published": "2023-08-22T19:16:20.617Z",
				"modified": "2023-08-24T22:03:03.697Z"
			},
			"ecosystems": [
				"cpe"
			]
		}
	],
	"detection": [
		{
			"ecosystem": "cpe",
			"criteria": {
				"operator": "OR",
				"criterias": [
					{
						"operator": "OR",
						"criterias": [
							{
								"operator": "OR",
								"criterias": [
									{
										"operator": "OR",
										"criterions": [
											{
												"vulnerable": true,
												"fix_status": {
													"class": "unknown"
												},
												"package": {
													"cpe": "cpe:2.3:a:dpic_project:dpic:2021-04-10:*:*:*:*:*:*:*"
												}
											}
										]
									}
								]
							}
						]
					}
				]
			}
		}
	],
	"data_source": {
		"id": "nvd-api-cve",
		"raws": [
			"vuls-data-raw-nvd-api-cve/2021/CVE-2021-33390.json"
		]
	}
}
